tink-docker/main.go

```diff
@@ -94,3 +96,33 @@ func DownloadFile(filepath string, url string) error {
 	_, err = io.Copy(out, resp.Body)
 	return err
 }
+
+func rebootWatch() {
+	fmt.Println("Starting Reboot Watcher")
+
+	// Forever loop
+	for {
+		if fileExists("/worker/reboot") {
```

Comment:
I assume some other process is creating this, so this is the signal to reboot?

Reply:
yep